packages/greenfield-app/pages/_app.tsx

```diff
@@ -20,22 +21,18 @@ import "../styles/globals.css";
 import { NetworthContextProvider } from "@popcorn/components/context/Networth";
 
 const bnb: Chain = {
-  id: 56,
-  name: "BNB Chain",
-  network: "bnb",
-  iconUrl: "https://assets.coingecko.com/coins/images/825/large/binance-coin-logo.png?1547034615",
-  rpcUrls: { default: "https://bsc-dataseed1.binance.org" },
-  blockExplorers: { default: { name: "BSCScan", url: "https://bscscan.com" } },
+  ...bsc,
+  rpcUrls: { default: { http: ["https://bsc-dataseed1.binance.org"] }, public: { http: ["https://bsc-dataseed1.binance.org"] } },
```

Comment:
if bsc is now supported we can get rid of the whole config here

Reply:
alright, I was just keeping it in incase we were using a different rpcURL from the default